src/javascript/app_2/App/Components/Elements/SettingsDialog/settings_dialog.jsx

```diff
@@ -0,0 +1,61 @@
+import classNames      from 'classnames';
+import React           from 'react';
+import PropTypes       from 'prop-types';
+import Tabs            from '../Tabs/tabs.jsx';
+import ChartSettings   from '../../../../Modules/Trading/Containers/SettingsDialog/settings_chart.jsx';
```

Comment:
@zakibinary 
Since the `SettingDialog` containers belong to the app and not just for the `Trading` modules it's better to move the `SettingsDialog` directory from `Trading/Containers/` to `App/Containers`